packages/bpk-svgs/tasks/getIconFontMetadataProvider.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import fs from 'fs';
+import path from 'path';
+
+export default (codepointsPath: string) => {
+  const absolutePath = path.join(process.cwd(), codepointsPath);
+
+  const codepoints = fs.existsSync(absolutePath)
+    ? JSON.parse(fs.readFileSync(absolutePath, 'utf8'))
+    : {};
+
+  return (file: string, callback: (err: ?Error, obj: ?Object) => mixed) => {
```

Comment:
Can this function get called many times and take the later branch? i.e when the codepoint isn't in the file already. If so can't the fact that the reading of the file is memoized cause a bug because we write over it?

Reply:
`codepoints` is read once and kept in memory so it's always up-to-date.